src/app/core/common-components/entity-form/entity-form.service.spec.ts

```diff
@@ -173,7 +173,7 @@ describe("EntityFormService", () => {
 
     schema.defaultValue = PLACEHOLDERS.NOW;
     form = service.createFormGroup([{ id: "test" }], new Entity());
-    expect(form.get("test")).toHaveValue(new Date());
+    expect(form.get("test").value).toEqual(new Date());
```

Comment:
Whats the difference here?

Reply:
the test sometimes failed although the date was exactly the same - I guess if MockDate was used before, it's not reference identical anymore?